crm_action/README.rst

```diff
@@ -0,0 +1,58 @@
+CRM Action
+==========
+
+This module was written to extend crm features.
+It delivers new object named "Actions" to follow history around leads and opportunities.
+This new object depends on ACL groups you give to users.
+
+Installation
+============
+
+To install this module, you just need to select the module and insure yourself dependencies are available.
+
+Configuration
+=============
+
+No particular configuration to use this module except :
+- potential customisation with type of actions,
+- add allowed user in the ACL group
+
+Usage
+=====
+
+To use this module, you need to :
+- create a lead or an opportunity
+- create a new action by using the button
+- you should get back and keep link between the models
+- you can overview all actions for any lead or opportunity with the new menu entry
+
+Known issues / Roadmap
+======================
+
+Credits
+=======
+
+Module developed and tested with Odoo version 8.0
+For questions, please contact our support services <support@savoirfairelinux.com>
```

Comment:
I also agree that we should keep neutrality in OCA modules, so I vote for removing this line.